test/JDBC/expected/linked_servers-vu-prepare.out

```diff
@@ -49,6 +49,18 @@ GO
 EXEC sp_addlinkedserver  @server = N'mssql_server3', @srvproduct=N'', @provider=N'tds_fdw', @datasrc=N'mssql_server2\ABC', @catalog=N'master'
 GO
 
+-- Create a linked server login with @useself = TRUE (Should throw error)
+EXEC sp_addlinkedsrvlogin @rmtsrvname = 'mssql_server', @useself = 'TRUE', @rmtuser = 'jdbc_user', @rmtpassword = '12345678'
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Only @useself = FALSE is supported. Remote login using user's self credentials is not supported.)~~
+
+
+-- Create a linked server login with @useself = FALSE. Also call sp_addlinkedsrvlogin from master.dbo schema
+EXEC sp_addlinkedsrvlogin @rmtsrvname = 'mssql_server', @useself = 'FALSE', @rmtuser = 'jdbc_user', @rmtpassword = '12345678'
+GO
```

Comment:
Add test case to test with default value, NULL value combination for various params.